library/qtest/src/Operations.qs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/// # Summary
+/// Runs a series of named test operations with expected results.
+///
+/// # Description
+/// Given an array of test names, test operations, and expected results (of the same type), returns `true`
+/// if all tests passed, and `false` if any one test failed.
+/// Prints the expected and received result for any failed tests.
+///
+/// # Input
+/// ## test_cases
+/// An array of five-arity tuples of the form `(test_name, num_qubits, qubit_prep_callable, callable_to_test, expected_result)`.
+/// `num_qubits` will be allocated in a qubit array and passed to `qubit_prep_callable` to prepare the state before executing 
+/// `callable_to_test`. Afterwards, `callable_to_test` will be called and its result will be compared to `expected_result`.
+///
+/// # Example
```

Comment:
Nit: Seems this example and one below needs to be changed since TestCases function of operations takes 5 arguments. It seems a copy paste typo from previous one.

Reply:
Nit: I cant see updated example here on Github, and I am unable to unresolve conversation, so commenting again to make sure it doesnt get lost

Reply:
PS: I realize that the function signature has been changed, it would be still good to have another toy example here to highlight that operation is being tested. Otherwise, someone might get confused by `Qtest.Functions.CheckAllTestCases` and `Qtest.Operations.CheckAllTestCases`. Also, it seems we need to update `() -> 42` to `() => 42`

Reply:
PS2: Seems PR was merged before I commented.